sof_wrapper/api/views.py

```diff
@@ -13,3 +14,71 @@ def add_header(response):
     response.headers['Access-Control-Allow-Origin'] = '*'
 
     return response
+
+
+@base_blueprint.route('/auditlog', methods=('OPTIONS', 'POST'))
+def auditlog_addevent():
+    """Add event to audit log
+
+    API for client applications to add any event to the audit log.  The message
+    will land in the same audit log as any auditable internal event, including
+    recording the authenticated user making the call.
+
+    Returns a json friendly message, i.e. {"message": "ok"} or details on error
+    ---
+    operationId: auditlog_addevent
+    tags:
+      - audit
+    produces:
+      - application/json
+    parameters:
+      - in: body
+        name: body
+        schema:
+          id: message
+          required:
+            - message
```

Comment:
Just to be pedantic, I think `level` is also `required`. We raise a 400 if it's absent

Reply:
Thanks for the detailed look! `level` is optional, we only raise a 400 if a bogus level is provided, using `info` by default if not sent.